tests/integration/test_records.py

```diff
@@ -367,3 +369,54 @@ def test_records_files_attached_correctly(app):
     record.commit()
 
     assert 'fulltext.pdf' in record.files
+
+
+@patch('inspirehep.modules.records.utils.open', mock_open())
+def test_records_files_are_populated_correctly_for_local_paths(app):
+    record_json = {
+        '$schema': 'http://localhost:5000/schemas/records/hep.json',
+        'document_type': [
+            'article',
+        ],
+        'titles': [
+            {'title': 'foo'},
+        ],
+        '_collections': ['Literature'],
```

Comment:
It doesn't look like the code requires the above data to run. Fixtures should be as minimal as possible, to avoid confusing the reader with irrelevant details.

Reply:
these are required from schema. Otherwise, it fails validation!